### ISSUE TITLE
Add more control over props in Asset Spray Tool

### ISSUE THREAD

**Issue by FullStack0verfl0w** [Facepunch/sbox-issues#2730]
### For?

Hammer

### What can't you do?

I want to set prop properties (like fade in/out, collision type, etc) in Asset Spray tool.

### How would you like it to work?

Maybe a drop down menu in Asset Spray Tool, where all the properties are.

### What have you tried?

Right now I have to select all the props I sprayed in my world and set these values by hand, which can cause a hang if I change values for a large set of props.

### Additional context

Also, I want to point out that the Asset Spray tool works slowly with every prop created in the world. This should be optimized.

**Comment by FireKeep-HLA** [comment]
I agree, using asset spray tool in Alyx was frustrating sometimes as I would have to manually go back and extra parameters for optimization purposes when it would've been much more easier to have just set the parameter before spraying them.
